src/com/xilinx/rapidwright/rwroute/GlobalSignalRouting.java

```diff
@@ -332,21 +332,19 @@ private static ClockRegion findCentroid(Net clk, Device device) {
 
     /**
      * Routes a static net (GND or VCC).
-     * @param currNet The current static net to be routed.
+     * @param pins A list of static pins to be routed.
```

Comment:
Are we assuming all the pins are on the same static net (all VCC or all GND)?  If so, perhaps indicate it in the Javadoc.